### PR TITLE
Add warning until we figure out api address

### PR DESCRIPTION
**Pull request by croomes**
Temporary workaround for setting api address

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,24 @@ $ cd storageos
 $ helm install .
 ```
 
+IMPORTANT:  The StorageOS api address must be manually set in the `api.address`
+value.  Set this to the ip address of one of the Kubernetes nodes, e.g. `http://10.0.0.1`.
```
Comment (oicheryl) [inline README.md]: http://10.0.0.1:**5705**

File: README.md [hunk 2]
```diff
@@ -15,6 +15,24 @@ $ cd storageos
...
+This introduces a single point of failure which we hope to remove by
+auto-populating a list of nodes or a service address.  The ip address (or
+hostname) given here MUST be accesible by the Kubernetes master, which may not
+be running the StorageOS service (`http://127.0.0.1` will work otherwise).
+
+example in values.yaml:
```
Comment (oicheryl) [inline README.md]: example => Example

File: README.md (hunk 1 above)
Comment (croomes) [reply README.md]: done, though the port is optional when it hasn't been changed

File: README.md (hunk 2 above)
Comment (croomes) [reply README.md]: Done

**Pull request merged**